packages/common/test/sites/_internal/SiteuUiSchemaSettings.test.ts

```diff
@@ -0,0 +1,55 @@
+import { buildUiSchema } from "../../../src/sites/_internal/SiteUiSchemaSettings";
```

Comment:
minor: looks like there's still a small typo in the file name? Should "SiteuUiSchemaSettings.test.ts" should be "SiteUiSchemaSettings.test.ts"?